javascript/javascript2/week1/homework-js2-week1/hyfBay/main.js

```diff
@@ -0,0 +1,35 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+console.log(products);
+
+const body = document.querySelector('body');
+const div = document.getElementsByClassName('products')[0];
+const resultsDiv = document.getElementsByClassName('results')[0];
+const input = document.querySelector('input');
+const searchButton = document.getElementsByClassName('search-button')[0];
+const ul = document.getElementsByClassName('products-list')[0];
+
+function update(element, elementToAdd) {
+  element.innerHTML = elementToAdd;
+}
+
+function renderProducts(products) {
+  products.forEach((product) => {
```

Comment:
check 'template string' which will help you to write complex HTML in an easy way or also you can take examples from others homeworks